app/views/proposals/_tooltip.html.haml

```diff
@@ -1,11 +1,12 @@
 - progress_status = event.progress_status
 %ul.list-unstyled
-  %li{'class'=>class_for_todo(progress_status['registered'])}
-    %span{'class'=>icon_for_todo(progress_status['registered'])}
-      - if progress_status['registered']
-        Speaker(s) registered to the conference
-      - else
-        = link_to 'Speaker(s) not registered to the conference', new_conference_conference_registration_path(event.program.conference.short_title)
+  - if @conference.registration_open? || (@conference.user_registered? current_user)
```

Comment:
We aren't checking if a user can register indeed. My point was that we should be checking that.

https://github.com/namangupta01/osem/blob/master/app/views/proposals/_tooltip.html.haml#L9 will redirect you to create a new registration, so the user needs to have the ability to create a new registration, otherwise it doesn't make sense to redirect users to a link they cannot access. Perhaps we should take that into consideration. What do you think?

https://github.com/namangupta01/osem/blob/master/app/views/proposals/index.html.haml#L77 needs to comply with the same conditions, so that we don't show that there are 3 missing items, but only show 2 missing items in the tooltip.